test/e2e/admin_portal.go

```diff
@@ -0,0 +1,198 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"                           //nolint
+	. "github.com/onsi/gomega"                           //nolint
+	. "github.com/serge1peshcoff/selenium-go-conditions" //nolint
+	. "github.com/tebeka/selenium"                       //nolint
+)
+
+var _ = Describe("Admin Portal E2E Testing", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+	var wdPoint *WebDriver
+	var wd WebDriver
+	var host string
+	JustBeforeEach(func() {
+		host, wdPoint = adminPortalSessionSetup()
+		wd = *wdPoint
+		wd.Get(host + "/v2")
+		wd.Refresh()
+	})
+
+	AfterEach(func() {
+		if wd != nil {
+			wd.Quit()
+		}
+	})
+	It("Should be able to populate cluster data correctly", func() {
+		err := wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		cluster, err := wd.FindElement(ByCSSSelector, "div[data-automation-key='name']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		Expect(cluster.Text()).To(Equal(os.Getenv("CLUSTER")))
+	})
+
+	It("Should be able to filter cluster data correctly", func() {
+		wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
+
+		filter, err := wd.FindElement(ByCSSSelector, "input[placeholder='Filter on resource ID']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		// Set filter so it doesn't match cluster name
+		filter.SendKeys("Incorrect Cluster")
+
+		wd.Wait(ElementIsLocated(ByID, "ClusterCount"))
+		text, err := wd.FindElement(ByID, "ClusterCount")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		Expect(text.Text()).To(Equal("Showing 0 items"))
+	})
+
+	It("Should be able to populate cluster info panel correctly", func() {
+		const CLUSTER_INFO_HEADINGS = 17
+		err := wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
```

Comment:
Should be an empty line above this, in order to separate the const declaration from the function call.

Reply:
Added